packages/chakra-components/src/MenuBar/menubar.js

```diff
@@ -114,6 +115,9 @@ const MenuBar = forwardRef(
           id={menuBarId}
           role={role}
           ariaLabel={ariaLabel}
+          display="flex"
+          flexDirection={mode && mode === "horizontal" ? "row" : "column"}
+          alignItems={mode && mode === "horizontal" ? "center" : "left"}
```

Comment:
Include these prop as style props object like 
```suggestion
let modeStyleProps = {}
if(mode==="vertical) {
   modeStyleProps: {
     flexDirection: column,
     alignItems: left
   } 
}
```

and pass this object after {...styleProps}

Reply:
done